libs/infinity_emb/infinity_emb/infinity_server.py

```diff
@@ -1,6 +1,7 @@
 import time
 from contextlib import asynccontextmanager
 from typing import Optional
+from fastapi.middleware.cors import CORSMiddleware
```

Comment:
Sorry, `import fastapi` is an optional package. Move import inside of the function below in this file with the other imports of fastapi.

Reply:
fixed it.